migtests/tests/import-file/run-import-file-test

```diff
@@ -70,6 +74,31 @@ main() {
 	import_data_file --data-dir ${TEST_DIR} --format csv --delimiter '\t' \
 			--file-table-map "accounts_data.csv:accounts" --has-header=true --batch-size 10000 --null-string "\N"
 
+	unset MAX_BATCH_SIZE_BYTES
```

Comment:
do you really need the unset, if you are calling export again?

Reply:
yeah, didn't realise it as I was doing it earlier for just one case. Made the change to remove all unsets, just kept it for the large file test case. Thanks